btree/btree.c

```diff
@@ -11091,6 +10788,57 @@ static void ut_mt_tree_oper(void)
 	btree_ut_num_threads_tree_oper(0);
 }
 
+static bool validate_nodes_on_be_segment(struct segaddr *rnode_segaddr)
+{
+	const struct node_type     *nt;
+	struct node_data {
```

Comment:
Declare structure outside of function body and use structure array in the function.

Reply:
This has been done in Motr code (especially in ut's) so that the structure is a part of only this function and is not used anywhere else.